pom.xml

```diff
@@ -56,13 +54,46 @@
                 </execution>
             </executions>
         </plugin>
+           <plugin>
+               <groupId>org.apache.maven.plugins</groupId>
+               <artifactId>maven-release-plugin</artifactId>
+               <version>2.5.1</version>
+           </plugin>
       </plugins>
     </build>
 
     <groupId>com.qubole.presto</groupId>
     <artifactId>udfs</artifactId>
-    <name>udfs</name>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.1.3-SNAPSHOT</version>
```

Comment:
I've already deployed. There is no way to test release plugin with SNAPSHOT
On Mon, 4 Jul 2016 at 21:01, Shubham Tagra <notifications@github.com> wrote:

> In pom.xml
> <https://github.com/qubole/presto-udfs/pull/13#discussion_r69504741>:
>
> >        </plugins>
> >      </build>
> >
> >      <groupId>com.qubole.presto</groupId>
> >      <artifactId>udfs</artifactId>
> > -    <name>udfs</name>
> > -    <version>0.1.0-SNAPSHOT</version>
> > +    <version>0.1.3-SNAPSHOT</version>
>
> shall we revert version to 0.1.0-SNAPSHOT? or have we already deployed to
> nexus staging so need to keep this version?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qubole/presto-udfs/pull/13/files/54c8f15119c5d0fd91da593281efc22f3fc74dde#r69504741>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABj_mldJpdPjLj1WDs6yw2-1taQfmEl4ks5qSdclgaJpZM4JDiqF>
> .
>